app/models/post.rb

```diff
@@ -1,5 +1,16 @@
 class Post < ActiveRecord::Base
-  belong_to :user
+  belongs_to :user, foreign_key: :author_id
   has_many :likes
   has_many :comments
+
+  def update_post_conuter
+    user.increment!(:post_counter)
```

Comment:
- [ ] Let's use Rails Active Callback `after_save` to call `update_post_counter` method every time a new post has been saved to database, then make `update_post_counter` method private. 


`after_save :update_post_counter`

  private

  def update_post_counter
       author.increment!(:posts_counter)
    end